ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -33,6 +33,12 @@ if(_source_files)
     list(APPEND _all_include_dirs ${ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS})
   endif()
 
+  # Language
+  set(_language "")
+  if(DEFINED ament_cmake_cppcheck_ADDITIONAL_LANGUAGE)
+    list(APPEND _language ${ament_cmake_cppcheck_ADDITIONAL_LANGUAGE})
```

Comment:
Doesn't the CMake function `ament_cppcheck` only accept a single language for `LANGUAGE`?

Reply:
cppcheck allows `c` or `c++` https://github.com/ament/ament_lint/blob/master/ament_cppcheck/ament_cppcheck/main.py#L68-L71

Reply:
> Doesn't the CMake function `ament_cppcheck` only accept **a single** language for `LANGUAGE`?

Reply:
Fixed https://github.com/ament/ament_lint/pull/239/commits/d751f081f63d6977526fbd07b863a92f5d220593

Reply:
The term "ADDITIONAL_LANGUAGE" doesn't make sense if only a single language can be selected.